lib/adal.js

```diff
@@ -316,6 +327,12 @@ AuthenticationContext.prototype._renewIdToken = function (callback) {
     this._loadFrame(urlNavigate, 'adalIdTokenFrame');
 };
 
+AuthenticationContext.prototype._urlContainsQueryStringParameter = function (name, url) {
+    name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+    var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
```

Comment:
Could you comment this a bit.  I don't understand what line 331 is doing.

Wouldn't the regex
> [?&]name=
Accomplish the same thing simpler?  The capture group is superfluous.  I think you can just use regex.test(url).